Wordle.API/Wordle.API/Controllers/ScoreController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Models;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers;
+[ApiController]
+[Route("[controller]")]
+public class ScoreController(LeaderboardService leaderboardService) : ControllerBase
+{
+    [HttpGet("Leaderboard")]
+    public async Task<List<Score>> GetTopTen()
+    {
+        return await leaderboardService.GetTopTenScores();
+    }
```

Comment:
```suggestion
    public async Task<List<PlayerDto>> GetTopTen()
    {
        return await leaderboardService.GetTopTenScores();
    }
```
In case you take suggeseted change for changing orginal `Score` into `PlayerDto`